pkgs/top-level/all-packages.nix

```diff
@@ -21491,7 +21491,7 @@ in
     inherit (gnome2) libIDL;
     inherit (rustPackages_1_38_0) cargo rustc;
     libpng = libpng_apng;
-    enableGTK3 = true;
```

Comment:
any reason for breaking this here? (I am not opposed)

Reply:
To keep it consistent with Firefox, I want to avoid the builds diverging in any nonessential way.